docs/topics/install/docker.rst

```diff
@@ -140,6 +135,81 @@ migrations::
 Gotchas!
 ~~~~~~~~
 
+Here's a list of a few of the issues you might face when using docker.
+
+Can't access the web server?
+----------------------------
+
+Check you've created an hosts file entry pointing ``olympia.dev`` to the
+relevant ip address.
+
+Also make sure you've copied ``docker-compose.override.yml.tmpl`` to
```

Comment:
Is there no way around this? It takes away from the turn-key `docker-compose pull && docker-compose up -d` setup. I didn't need to do this on my Mac and I hit `127.0.0.1` just fine...